src/d3a/setup/kpi/d3asim_2103_nopv1.py

```diff
@@ -13,8 +13,8 @@ def get_setup(config):
 
     ConstSettings.GeneralSettings.DEFAULT_UPDATE_INTERVAL = 1
     ConstSettings.IAASettings.MARKET_TYPE = 1
-    ConstSettings.IAASettings.MIN_OFFER_AGE = 0
-    ConstSettings.IAASettings.MIN_BID_AGE = 0
+    # ConstSettings.IAASettings.MIN_OFFER_AGE = 0
+    # ConstSettings.IAASettings.MIN_BID_AGE = 0
```

Comment:
why are these commented out?

Reply:
This is related to [this](https://github.com/gridsingularity/d3a/pull/1261#discussion_r742843421) comment. Having a bid and offer age of 0 is a bit extreme and was probably added in order to bend the behaviour of the setup so that the expected test results are met. We used the default values now (which is 2). I will remove the comments though.